### DISCUSSION TITLE
Documentation for inputs under inputrunner

### DISCUSSION THREAD

**Discussion by ruflin** [elastic/assetbeat#22] [General]
The inputs built under input runner will require docs. Where should these docs go?

**Comment by ruflin** [comment]
@eyalkraft How does cloudbeat deal with this challenge?

**Comment by eyalkraft** [comment]
So far cloudbeat's docs are maintained under [security-team/docs/cloud-security-posture-team/](https://github.com/elastic/security-team/tree/main/docs/cloud-security-posture-team) and readable through docs.elastic.dev (for example [cloudbeat design](https://docs.elastic.dev/security-solution/cloud-security-posture/cloudbeat/cloudbeat-design)). but that's not ideal as it's a private repository, and it's not coupled with cloudbeat versions as it would ideally be.

**Comment by ruflin** [comment]
Does this mean there are no docs for users available for the integration on https://www.elastic.co/guide/index.html outside of Fleet?

**Comment by eyalkraft** [comment]
We have [KSPM docs](https://www.elastic.co/guide/en/security/8.6/kspm.html), and I'm pretty sure @tinnytintin10 is also making sure we have some docs for CSPM which we're releasing with 8.7 

Here it is:
- https://github.com/elastic/security-docs/issues/2788
